.travis.yml

```diff
@@ -15,5 +15,5 @@ install:
 
 script:
   - sleep 20
-  - docker exec -it --user=solr my_solr bin/solr create_collection -c bioentities-v1
```

Comment:
We are not testing anymore the bioentities creation?

Reply:
or where is the bioentities one created now?

Reply:
creating collection with `gxa-analytics-v1`  https://github.com/ebi-gene-expression-group/index-gxa/pull/3/files#diff-354f30a63fb0907d4ad57269548329e3R18

I don't see bioentities-v1 in the latest commit.

Reply:
We need to have the bioentities testing that we are removing here.

Reply:
bioentities tests are there and succeeds. Even we create `gxa-analytics-v1` while loading in schema bioentities collection gets created. I've tested this one on `localhost:8983`